packages/js/github-actions/actions/prepare-extension-release/README.md

```diff
@@ -40,7 +40,13 @@ on:
         with:
           version: ${{ github.event.inputs.version }}
           type: ${{ github.event.inputs.type }}
-          wp_version: ${{ github.event.inputs.wp_version }}
-          wc_version: ${{ github.event.inputs.wc_version }}
-          main_branch: 'trunk'
+          wp-version: ${{ github.event.inputs.wp-version }}
+          wc-version: ${{ github.event.inputs.wc-version }}
```

Comment:
I can see this matches the inputs in the original workflow:

```
          wp-version: ${{ github.event.inputs.wp-version }}
          wc-version: ${{ github.event.inputs.wc-version }}
```

But in the final simulate / release commands the versions are missing:
```
woorelease simulate --product_version=1.2.3  --generate_changelog https://github.com/mikkamp/tryout/tree/release/1.2.3
```

Is a dash in a variable name allowed? From a quick search I'm seeing the following might be needed:
```
${{ github.event.inputs['wp-version'] }}
```

Reply:
I tried it in https://github.com/tomalec/grow/blob/b1448e10c86b1e4460bf2e9974911ab89f9cd692/.github/workflows/extension-release.yml#L19-L41
and it seems to work correctly

https://github.com/tomalec/grow/pull/35

could you make your repo public?

Reply:
Maybe it's better to change to camelCase, so it will work for sure and will not require mapping in `js` files.

Or we can change to the snake_case, to follow GitHub workflow convention (which is still a bit inconsistent and conducting to me as it [has `branches-ignore`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-branches))

Reply:
Are you sure you used 
```yml
uses: woocommerce/grow/prepare-extension-release@actions-v1.6.1-pre
```
not `v1`?

I tested in on a fresh repo and it works fine https://github.com/tomalec/test-dashes-in-workflows/actions/runs/5265434320

Reply:
It would help if I read the full testing instructions. I was originally going by the steps here: https://github.com/woocommerce/google-listings-and-ads/pull/1982

And assumed you were going to update the branch there when the PR got merged, so I didn't double check. Switching it out to the branch allows everything to work. I'll go everything again.